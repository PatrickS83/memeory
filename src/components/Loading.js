import React from 'react';

const Loading = () => (
  <section className="section is-medium has-text-centered">
    <div className="sk-folding-cube">
      <div className="sk-cube1 sk-cube" />
      <div className="sk-cube2 sk-cube" />
      <div className="sk-cube4 sk-cube" />
      <div className="sk-cube3 sk-cube" />
    </div>
    <h2>Packaging cards ...</h2>
  </section>
);

export default Loading;
