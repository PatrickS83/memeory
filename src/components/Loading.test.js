import React from 'react';
import { shallow } from 'enzyme';
import Loading from './Loading';

describe('<Loading/>', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<Loading />);
    expect(wrapper.length).toEqual(1);
  });

  it('renders the Loading Component as saved in snapshot', () => {
    const wrapper = shallow(<Loading />);
    expect(wrapper).toMatchSnapshot();
  });
});
