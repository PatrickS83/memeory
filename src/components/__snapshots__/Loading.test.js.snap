// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading/> renders the Loading Component as saved in snapshot 1`] = `
<section
  className="section is-medium has-text-centered"
>
  <div
    className="sk-folding-cube"
  >
    <div
      className="sk-cube1 sk-cube"
    />
    <div
      className="sk-cube2 sk-cube"
    />
    <div
      className="sk-cube4 sk-cube"
    />
    <div
      className="sk-cube3 sk-cube"
    />
  </div>
  <h2>
    Packaging cards ...
  </h2>
</section>
`;
